src/public/video.ts

```diff
@@ -78,7 +109,7 @@ export namespace video {
   /**
    *  Video effect change call back function definition
    */
-  type VideoEffectCallBack = (effectId: string | undefined) => void;
+  type VideoEffectCallBack = (effect: string | undefined, effectParam?: string) => void;
```

Comment:
Is there any reason to change 'effectId' to 'effect? For video app, the parameter's value is still effect Id, if changed to effect may need 3rd party developer update its code when using new sdk.

Reply:
Fixed